service/controller/aws/v14/version_bundle.go

```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "cluster-operator",
+				Description: "Added support for setting the kubeconfig with the app-operator-api certs.",
```

Comment:
```suggestion
				Description: "Added support for creating a kubeconfig for app-operator.",
```

This is pretty close and writing good changelogs is tricky. The changelogs are customer facing so mentioning the app-operator-api is a bit too low level.

Reply:
Oh also please change the type to `versionbundle.KindAdded`. Thx!